SDWebImage/SDWebImageDownloader.m

```diff
@@ -192,6 +192,37 @@ - (void)setOperationClass:(nullable Class)operationClass {
     }
 }
 
+- (id<SDWebImageOperation>)loadImageWithURL:(NSURL *)url options:(SDWebImageOptions)options progress:(SDWebImageLoaderProgressBlock)progressBlock completed:(SDWebImageLoaderCompletedBlock)completedBlock context:(SDWebImageContext *)context {
```

Comment:
Code moving as well. Actually the `SDWebImageDownloader`'s basic logic does not change at all.